cmd/auth-rest/startcmd/start.go

```diff
@@ -143,6 +149,7 @@ type authRestParameters struct {
 	tlsParams       *tlsParams
 	oidcParams      *oidcParams
 	bootstrapParams *bootstrapParams
+	staticFiles     string
```

Comment:
`staticFiles` is not being assigned a value

Reply:
done

Reply:
done